src/platforms/common/profiling/index.mdx

```diff
@@ -159,6 +183,44 @@ The <PlatformIdentifier name="profiles_sample_rate" /> setting is *relative* to
 
 </PlatformSection>
 
+
+<PlatformSection supported={["rust"]}>
+
+<Note>
+
+Rust Profiling alpha is available starting in SDK version `0.29.0`.
+
+</Note>
+
+In order to use Profiling you first need to enable the `profiling` feature in the `sentry` dependency inside the project `Cargo.toml`:
+
+```cargo
```

Comment:
@imatwawana where can I check which syntax (`xml`, `python`, etc.) is supported for these code snippets?

I've put `cargo` here but I'm unaware of whether that is actually supported or not.

ps. thanks for the previous feedback.

Reply:
> @imatwawana where can I check which syntax (`xml`, `python`, etc.) is supported for these code snippets?
> 
> I've put `cargo` here but I'm unaware of whether that is actually supported or not.
> 
> ps. thanks for the previous feedback.

I'm not sure where you'd check that for the whole repository. @untitaker do you know? But for Rust, `cargo` is definitely already in use in this repo. Also you can check the preview generated by Vercel to see if it's rendering the way you intend. Here's [that page](https://sentry-docs-git-viglia-rustprofiling.sentry.dev/platforms/rust/profiling/) in the preview so you can confirm. When you're in the preview, you can navigate to any page, but you can't use the search or you'll end up back in the live docs.